Dribbblish/user.css

```diff
@@ -894,9 +894,7 @@ section.contentSpacing {
 
 /* buddy feed w/ hidden text*/
 .buddyFeed-hide-text .NdQkQZhcYIEcJnRdAYcQ,
-.buddyFeed-hide-text .main-buddyFeed-header {
-    display: none;
-}
+.buddyFeed-hide-text .main-buddyFeed-header

```

Comment:
think you made an error here, is this supposed to have a style?

Reply:
After doing some trial and error, I found that having `display: none;` in the code caused the entire Spotify window to shift left after opening lyrics (as seen in the image below). I removed it and everything seemed to work like a miracle.

![bug](https://user-images.githubusercontent.com/67120455/231038832-738f1313-9223-42a3-b411-19a17d41d4e1.png)

Due to my extremely limited knowledge on CSS, I have no idea why or how this one line of code causes such a major change in the UI, or if I unintentionally changed another important functionality along the way, but what I did seemed to fix the issue and I haven't encountered any more issues thus far with the UI.